### ISSUE TITLE
A change users want.

### ISSUE THREAD

**Issue by 07nainankit** [ApoorvaRajBhadani/cf-analytics#2]
currently we can just get the graph about number of questions that are solved by a particular user of a particular rating like 10 of 800 rating and so on.

what we are requesting is along with this number can we get the list of problems solved by any user of all difficulties. 

it will be really beneficial for practicing.

**Comment by 07nainankit** [comment]
like ankit nain 
800
ques 1 
ques 2 
........
900
ques 1 
ques 2 
......
and so on

**Comment by ApoorvaRajBhadani** [comment]
What I think is this will make the profile page look cluttered. Wouldn't it?

Or do you want something like this - https://github.com/ApoorvaRajBhadani/cf-analytics/pull/1

What I would suggest, is you make a pull request with the feature you want then I will review it.
Or elaborate more on what you want.

**Comment by 07nainankit** [comment]
It will be manageable it will be helpful for beginners like me and many
people want it even their is a blog about it.
The question list will definitely act as a guide for an individual who have
shortage of time like me who cannot explore codeforces because of shortage
of time.

Well if clustering is an issue then we can just have a click feature like
if I want to see which 800 rating ques a x person have done then I will
click on 800 and a separate page or same page a list appears it will not
hinder user experience but it will be rather very helpful.

Thanks bhiya

**Comment by 07nainankit** [comment]
![sample ](https://github.com/ApoorvaRajBhadani/cf-analytics/assets/98472907/3402fca0-f33d-4c0a-bd84-e011bdfa71bb)

like here if we go to submissions we can see all the submissions from the start to the current.
what i am saying is it can we get it organized like rating wise list of ques.
like currently CF ANALYTICS provide us the number but can it be done that we get list of questions too.

**Comment by 07nainankit** [comment]
> What I think is this will make the profile page look cluttered. Wouldn't it?
> 
> Or do you want something like this - #1
> 
> What I would suggest, is you make a pull request with the feature you want then I will review it. Or elaborate more on what you want.

yes like #1 just one thing the problem set should have the ques that are just solved by the user who's profile we have clicked that's what exactly needed.

**Comment by ApoorvaRajBhadani** [comment]
Hi please join slack we'll discuss there

https://join.slack.com/t/cfanalyticsdi-nkf8468/shared_invite/zt-25h8xpxd3-eKatqZUJ3~EghUO_OA01ZQ
